src/Microsoft.IdentityModel.Tokens/SecurityAlgorithms.cs

```diff
@@ -108,7 +108,7 @@ public static class SecurityAlgorithms
         public const string RsaSsaPssSha384 = "PS384";
         public const string RsaSsaPssSha512 = "PS512";
 
-        // See : https://tools.ietf.org/html/rfc7518#section-5.1
+        // See : https://datatracker.ietf.org/doc/html/rfc7518#section-5.1
```

Comment:
nit: extra space

Reply:
It is good to be consistent with spacing and casing.